src/styles/main.scss

```
@@ -1,3 +1,4 @@
-@import "blocks/page";
-@import "blocks/header";
-@import "blocks/catalog";
+@import "./utils/variables.scss";
+@import "./blocks/page.scss";
```

Only one small suggestion
Put an empty line between util files and block files

```suggestion
@import "./utils/variables.scss";

@import "./blocks/page.scss";
```